iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -424,4 +578,152 @@ public static void checkLossFactor(Validable validable, float lossFactor) {
             throw new ValidationException(validable, "loss factor must be >= 0 and <= 100");
         }
     }
+
+    private static ValidationLevel checkRtc(Validable validable, RatioTapChanger rtc, Network network, boolean throwException, Reporter reporter) {
+        ValidationLevel validationLevel = ValidationLevel.LOADFLOW;
+        if (rtc.getTapPosition().isEmpty()) {
+            throwExceptionOrLogError(validable, "tap position is not set", throwException, reporter);
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.STATE_ESTIMATION);
```

Comment:
I don't getwhy we need a min...

Reply:
It doesn't really matter today as we only have two validation level, but if we have others later, it allows us to only keep the lowest validation level between the current one and the one computed from the check